gotfparse/cmd/tftest/main.go

```diff
@@ -18,15 +18,31 @@ func main() {
 
 	path := os.Args[1]
 	tfd, err := converter.NewTerraformConverter(path)
-	if err != nil {
-		fmt.Print(err)
-		return
-	}
+	checkError(err)
+
 	data := tfd.VisitJSON().Data()
+
 	j, err := json.MarshalIndent(data, "", "\t")
-	if err != nil {
-		fmt.Print(err)
+	checkError(err)
+
+	fmt.Print(string(j))
+
+	flags := os.O_CREATE | os.O_APPEND | os.O_TRUNC | os.O_RDWR
+	f, err := os.OpenFile("output.json", flags, 0o666)
```

Comment:
Make this an explicit flag or remove it. Expecting to have to redirect the output using `>` is reasonable for this kind of tool. I would be annoyed if I ran this and piped it to grep and it produced an `output.json` for no reason.

Reply:
Nope, you're right, this was silly and lazy. Removing it.